bigtop-packages/src/common/tez/do-component-build

```diff
@@ -33,4 +33,11 @@ BUILD_TEZ_OPTS="clean package \
 -Phadoop28 \
 -DskipTests"
 
+# install tez-ui by bower need the root permissions in openEuler
```

Comment:
You may have a misconception about Bower installation here.
It's not allowed to install `bower` by user: `root`.

The `--allow-root `flag allows the installation of Bower by the `root` user.
But we should build and test Bigotp components by `jenkins` user.
`--allow-root` should be false not be true here.
Could you please try to build tez by jenkins user?

Reply:
@guyuqi 
Hi, we need to grant the root permission to the jenkins user to run the bower installation commands in the openEuler system.
![tez-error](https://github.com/apache/bigtop/assets/113407588/1e923013-3c46-4f9a-ba20-0ea9402286c3)

Reply:
It seems you actually were building `Tez` by root users from the screenshot above:  `/root/bigtop-fedora/build`

Reply:
> It seems you actually were building `Tez` by root users from the screenshot above: `/root/bigtop-fedora/build`

@guyuqi 
If the Jenkins user is not granted the root permission to run the Bower installation command in the openEuler system, the error will occur. the screenshot above: /root/bigtop-fedora/build is the specified working directory.
![image](https://github.com/apache/bigtop/assets/113407588/5b894313-7d4a-4b2c-96fc-a442d4ec5843)

My build command is :
![image](https://github.com/apache/bigtop/assets/113407588/f98f90d0-918b-429e-af5d-3fdcacaf4299)

Reply:
The log tells:  "Since bower is a user command, there is no need to execute it with superuser permissions".
Please try to add -u jenkins in build command:
`docker run --rm -u jenkins ....`

Reply:
The TEZ building has been verified on the other Distros (Ubuntu, Fedora, Rocky).

Reply:
jenkins uid is 1000, not 1001:
```
jenkins@e52ab97e4481:/ws$ id
uid=1000(jenkins) gid=1000(jenkins) groups=1000(jenkins)
```

Reply:
> The TEZ building has been verified on the other Distros (Ubuntu, Fedora, Rocky).

@guyuqi 
I tried to grant 1000 uid to the specified working directory, but still get this error. The root permission is required for the openEuler system to run the bower installation command. However, the Jenkins user does not have the root permission in the openEuler system.

Reply:
It smoke test successful without this modify code using **_jenkins_** way you gave us.
![image](https://github.com/apache/bigtop/assets/11262916/030a5749-8519-4169-ba90-bfebda6eb456)

This link help us: https://cwiki.apache.org/confluence/display/BIGTOP/Bigtop+CI+Setup+Guide

I will close the PR.
@guyuqi

Reply:
Thanks your help @guyuqi